### PR TITLE
Feature/259 update concatenated labels

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [EQ-259](https://jira.epa.gov/browse/EQ-259)

## Main Changes:
* Updated option labels that consist of ID and Name fields to take the format `ID **(Name)**`.
* Updated the field labels to match the same format.
  * I didn't update the label of the _State_ field, because it still refers to one underlying column.
* Added a semibold font-weight. This font variant _is_ included in USWDS, but the font-face isn't specified by default. Rather than referring to the font file in the USWDS `node_modules` path, I thought it would be safer to copy the file to the public folder.
* Moved `styles.css` to the public folder. This is done simply to simplify the path described above and to keep assets in one place. Also, if we ever choose to compile USWDS as [described in its documentation](https://designsystem.digital.gov/documentation/getting-started/developers/phase-two-compile/), the output would be placed there. 

## Steps To Test:
1. Run the ETL locally.
2. Start the app, and go to http://localhost:3000/attains/assessments.
3. Confirm that options in the _State_ field display correctly when no options are selected, and after selecting a region.
4. Confirm options of other ID/Name fields display correctly.

### CODE REVIEW

**Comment by maxdiebold-erg** [comment]
@cschwinderg I added the bold **(Names)** like you suggested. After trying it out, we may actually want to make the IDs bold instead, since it can get a bit busy as it is now, but we can see what the ATTAINS crew thinks.

File: app/client/src/config/fields.ts [hunk 1]
```diff
@@ -271,7 +271,8 @@ export const filterFields = [
   },
   {
     key: 'actionId',
-    label: 'Action ID',
+    label: 'Action ID (Action Name)',
+    placeholder: 'Select an Action',
```
Comment (cschwinderg) [inline app/client/src/config/fields.ts:275]:
```suggestion
    placeholder: 'Select an Action...',
```

I think these should have the `...` to be consistent with the other drop downs or the `...` should be removed from the other drop downs.

File: app/client/src/routes/home.tsx [hunk 2]
```diff
@@ -728,6 +731,16 @@ function SelectFilter<
...
+      formatOptionLabel={(option) =>
+        secondaryFilterKey ? (
+          <div>
+            {option.value}{' '}
+            <span style={{ fontWeight: 600 }}>({option.label})</span>
+          </div>
+        ) : (
+          option.label
+        )
+      }
```
Comment (cschwinderg) [inline app/client/src/routes/home.tsx:743]: Can you fix the codesmell here? Basically, this just needs to be put into a separate function component.

**Review by cschwinderg — CHANGES_REQUESTED** [review]
Looks good!

I agree with you. I think it would look better to have the ids bold. 

I left a couple of minor suggestions below.

File: app/client/src/config/fields.ts (hunk 1 above)
Comment (maxdiebold-erg) [reply app/client/src/config/fields.ts:275]: Good catch!

File: app/client/src/routes/home.tsx (hunk 2 above)
Comment (maxdiebold-erg) [reply app/client/src/routes/home.tsx:743]: Yeah, the first way I tried didn't fix the smell, I'll try another way.

Comment (maxdiebold-erg) [reply app/client/src/routes/home.tsx]: I used a callback to fix this. I know SonarCloud advises against using a callback, but the prop in question is actually looking for a callback in this case.

**Review by cschwinderg — APPROVED** [review]
Thanks for making those changes!

**Pull request merged**